lib/nats/io/jetstream/manager.rb

```diff
@@ -58,6 +58,25 @@ def stream_info(stream, params={})
         JetStream::API::StreamInfo.new(result)
       end
 
+      # edit_stream edits an existed stream with a given config.
+      # @param config [JetStream::API::StreamConfig] Configuration of the stream to create.
+      # @param params [Hash] Options to customize API request.
+      # @option params [Float] :timeout Time to wait for response.
+      # @return [JetStream::API::StreamCreateResponse] The result of creating a Stream.
+      def edit_stream(config, params={})
```

Comment:
Let's change to `update_stream` so that it has similar signatures to the Go client? https://github.com/nats-io/nats.go/blob/b142f6053bcff1167f6b6c5fc1a56df120fd059f/jsm.go#L32
```suggestion
      def update_stream(config, params={})
```